crypto/x509v3/v3_cpols.c

```diff
@@ -413,7 +413,8 @@ static void print_notice(BIO *out, USERNOTICE *notice, int indent)
             num = sk_ASN1_INTEGER_value(ref->noticenos, i);
             if (i)
                 BIO_puts(out, ", ");
-            tmp = i2s_ASN1_INTEGER(NULL, num);
+            if ((tmp = i2s_ASN1_INTEGER(NULL, num)) == NULL)
+                return;
```

Comment:
There are multiple reasons why i2s_ASN_INTEGER can fail - either the input is invalid (there are various checks) or you are in an out-of-memory condition. I don't know that "InvalINT" is a reasonable thing to output here. 
How do we handle other errors on the input in printing functions in general?